cli/types/cypress.d.ts

```diff
@@ -657,6 +657,20 @@ declare namespace Cypress {
      */
     as(alias: string): Chainable<Subject>
 
+    /**
+     * Attach a file to the given <input> element, or drag and drop a file over any subject.
```

Comment:
```suggestion
     * Select a file to upload to the given <input> element, or drag and drop a file over any subject.
```

Reply:
Applied slightly different changes in same spirit.